src/Compilers/CSharp/Portable/Symbols/Source/SourceNamedTypeSymbol_Bases.cs

```diff
@@ -156,9 +156,12 @@ protected override void CheckInterfaces(DiagnosticBag diagnostics)
 
                             if (other.Equals(@interface, TypeCompareKind.IgnoreNullableModifiersForReferenceTypes))
                             {
-                                diagnostics.Add(ErrorCode.WRN_DuplicateInterfaceWithNullabilityMismatchInBaseList, location, @interface, this);
+                                if (!other.Equals(@interface, TypeCompareKind.ObliviousNullableModifierMatchesAny))
```

Comment:
>if (!other.Equals(@interface, TypeCompareKind.ObliviousNullableModifierMatchesAny)) [](start = 32, length = 83)

Should we still report a warning for an oblivious mismatch?

Reply:
I don't think so. That seems the most likely/common/useful scenario: you have a base type with oblivious from a library, then your type implemented the interface with specific nullability.

---
In reply to: [324916964](https://github.com/dotnet/roslyn/pull/38563#discussion_r324916964) [](ancestors = 324916964)